cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1385,6 +1385,7 @@ public void testReconcile(Params params, VertxTestContext context) {
 
         Checkpoint fooAsync = context.checkpoint();
         Checkpoint barAsync = context.checkpoint();
+        Checkpoint completeTest = context.checkpoint();
```

Comment:
Moved checkpoint creation higher, as previously this checkpoint did nothing after `fooAsync` and `barAsync` were flagged.